components/scream/src/share/tests/kokkos_utils_tests.cpp

```diff
@@ -1,10 +1,11 @@
 #include "catch2/catch.hpp"
 
 #include "share/util/scream_kokkos_utils.hpp"
+#include "share/util/scream_arch.hpp"
 
 namespace {
 
-TEST_CASE("data_type", "") {
+TEST_CASE("data_type", "[kokkos_utils]") {
```

Comment:
The catch2 tutorial has tags within square braces, not if that's a convention worth following.